packages/@aws-cdk/aws-lambda-event-sources/README.md

```diff
@@ -0,0 +1,106 @@
+## AWS Lambda Event Sources
+
+This module includes classes that allow using various AWS services as event
+sources for AWS Lambda via the high-level `lambda.addEventSource(source)` API.
```

Comment:
I think in most other places where invert the style of control. For example, we have CloudWatch Events `.addTarget()` I believe. To me it also seems more logical to say `resource.runLambda(...)`. Why this way around?

(To put it in programming terms, it seems like the difference between GOTO and [COMEFROM](https://en.wikipedia.org/wiki/COMEFROM), and the latter one is less readable)

Reply:
As the readme mentions, this is an alternative API for the resource-centric one. The reason I believe it is useful is because this is how "serverless people" think, which are lambda-centric. This is also the semantics used by Lambda across the board (both [documentation](https://docs.aws.amazon.com/lambda/latest/dg/invoking-lambda-function.html), console, [SAM](https://github.com/awslabs/serverless-application-model/blob/master/versions/2016-10-31.md#event-source-types), etc).

This doesn't replace the resource-centric APIs, but just adds a high-level library for people who prefer to think about the world from this perspective, which I believe is a legitimate mental model.